src/frontend/views/Youtube/YoutubeTemplate.tsx

```diff
@@ -93,8 +93,9 @@ export function YoutubeTemplate({
 
       {status === 'urlReady' && (
         <p className={styles.buttonsLine}>
-          <a className={styles.ctaButton} href={authUrl}>
-            Start Credential process
+          <a className={styles.signinButton} href={authUrl}>
```

Comment:
```suggestion
          <a
            className={styles.youtubeButton}
            href={authUrl}
            aria-label="Sign in with Youtube"
          >
```

Reply:
The current design is not accessible, since screen readers will simply pick up a button that says 'Sign in with'. We can add the aria-label attribute to make it clear what the button is doing for these users.